src/app/home/navigation.component.ts

```diff
@@ -7,13 +7,28 @@ import { Router } from '@angular/router';
   selector: 'planet-navigation',
   template: `
     <ul>
+      <li [ngStyle]="{'width': '10%'}">
+        <i *ngIf="selectedIcon"><img [src]="selectedIcon" alt="selectedIcon" style="margin-bottom: -15px"></i>
+        <mat-form-field [ngStyle]="{'width': '60%'}">
+          <mat-select [(ngModel)]="selectedLanguage" (change)="optionSelected($event)" [(value)]="selected">

```

Comment:
Value selected needs to be done. When flag is selected it doesn't reflect LTR and RTL direction as expected but works on reload page. So, it is in progress.